api/src/identity-access-management/domain/services/cnfpt-oidc-authentication-service.js

```diff
@@ -0,0 +1,13 @@
+import { OidcAuthenticationService } from './oidc-authentication-service.js';
+
+const CNFPT_CLAIMS_MAPPING = {
+  firstName: ['given_name'],
+  lastName: ['name', 'family_name'],
```

Comment:
D'après les informations actuelles dont nous disposons, il y a toujours un `name` renseigné pour chaque utilisateur avec la valeur que nous attendons côté Pix. C'est actuellement une mauvaise utilisation du claim `name` mais il y a toujours la valeur que nous attendons. Alors que dans `family_name` il n'y a pas la valeur que nous attendons (et ce claim est également actuellement mal utilisé). Donc faisons reflété dans le code l'état de notre connaissance sur la situation : 

```suggestion
  lastName: ['name'],
```